### ISSUE TITLE
Gizmo offset should be proportional to Dimension size

### ISSUE THREAD

**Issue by rulon** [kevancress/MeasureIt-ARCH#230]
Is there any distance setting between "gizmo" arrow and according dimension?
I use arrow for drag drop
But at the moment, the margin from the corresponding measurement is too large.

![image](https://user-images.githubusercontent.com/499425/183633048-daa7ca04-a811-4257-a52f-98303957b1f5.png)

**Comment by kevancress** [comment]
@rulon Yeah this has been bugging me for a while too. It's just hardcoded right now, but I should make it proportional to the dimensions width so that it still works at small and large scales...

**Comment by kevancress** [comment]
Alright so I added a basic scaling of the offset based on the dimensions length... e22fa1b5d3102cf39f966deb545f83bb4383606b

This should help, but isn't the best... Going to leave this issue open till I find a better solution

**Comment by rulon** [comment]
Is it possible to make the dimension move when this text is dragged by mouse?

![image](https://user-images.githubusercontent.com/499425/184339542-5fa04b74-167a-406c-ae1b-21bad240ad7e.png)

**Comment by kevancress** [comment]
@rulon 

Yeah I think it should be possible, but would require a fairly significant overhaul of the Gizmo and dimension code...

I'll make a separate thread for it as a feature request so I can keep it in mind
